woocommerce/woocommerce
# Update the pnpm constraint to 7.12.2+ now auto-install is fixed.

### Changes proposed in this Pull Request:

As of https://github.com/pnpm/pnpm/pull/5394 being merged and released in https://github.com/pnpm/pnpm/releases/tag/v7.12.2 we no longer need to fix the pnpm version so specifically.

This allows the use of 7.12.2 and upwards.

### How to test the changes in this Pull Request:

1. See CI pass
2. Locally install latest pnpm `npm i -g pnpm`
3. install and build `pnpm i && pnpm run build`

### Other information:




### FOR PR REVIEWER ONLY:
